security-center/snippets/package.json

```diff
@@ -14,7 +14,8 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@google-cloud/pubsub": "^4.0.0",
-    "@google-cloud/security-center": "^8.7.0"
+    "@google-cloud/security-center": "^8.7.0",
+    "@google-cloud/bigquery": "^7.0.0"
```

Comment:
question: Is the bigquery client library only used for testing setup?

suggestion: Move @google-cloud/bigquery to devDependencies.

Reply:
Addressed.